src/main/java/unicash/logic/commands/SummaryCommand.java

```diff
@@ -20,9 +20,14 @@ public class SummaryCommand extends Command {
 
     public static final String SHOWING_SUMMARY_MESSAGE = "Opened UniCa$h summary window.";
 
+    public static final String NOT_SHOWING_SUMMARY_MESSAGE = "You have no expenses to summarize.";
```

Comment:
Believe these should be standardized to use @jamz903 's new success message system. If you have an alternative case, can just declare a separate constant like in #137 and also good to standardise the naming to `MESSAGE_...` since that's how the other commands do it

Reply:
Fixed!